javascript/javascript1/week3/homework1/script.js

```diff
@@ -0,0 +1,22 @@
+
+
+const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+  };
+  
+
+
+let horas = travelInformation.destinationDistance / travelInformation.speed;
```

Comment:
Looks good. Let's make it a function and check if any variables could be declared const.